pkg/mqtrigger/messageQueue/nats/nats.go

```diff
@@ -57,7 +57,8 @@ func init() {
 }
 
 const (
-	natsProtocol          = "nats://"
+	// Commenting it to ensure verify-staticcheck passes
```

Comment:
No need for this comment in code, the PR explains it. Same in other file

Reply:
Done